src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
```

Comment:
- [ Required ] I think it would be nice if you imported the `categories` reducer the same way you've imported the `bookReducer` into this file. Also please remember to include it in the `redux-store` to make it easy for the app to interact with the categories reducer. Please see the example below for a suggestion on how to achieve this.

## Example code
```ruby
import { configureStore } from '@reduxjs/toolkit';
import bookReducer from './books/booksSlice';
import categoriesReducer from './categories/categoriesSlice';

const store = configureStore({
  reducer: { books: bookReducer, categories: categoriesReducer },
});

export default store;

```

Reply:
@Felix45  Thank you for your feedback. It helped me a lot.👍🏼